From c416b1de5a54ea83d29d99261324d8a4a403bc64 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 26 Jan 2012 17:30:06 +0100
Subject: [PATCH] fix kernel modules search for s390

find_kernel_modules_by_path only takes one parameter

fixes c254ac796fe52d19ad47df99055cd58fee5e517e
and 4fcd5409e09e693349e991bd2a78303114a735f5
---
 modules.d/40network/module-setup.sh   |    3 ++-
 modules.d/90multipath/module-setup.sh |    2 +-
 modules.d/95iscsi/module-setup.sh     |    2 +-
 3 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/modules.d/40network/module-setup.sh b/modules.d/40network/module-setup.sh
index 9017413..384baf1 100755
--- a/modules.d/40network/module-setup.sh
+++ b/modules.d/40network/module-setup.sh
@@ -58,7 +58,8 @@ installkernel() {
         [[ $debug ]] && set -x
     }
 
-    find_kernel_modules_by_path drivers/net drivers/s390/net | net_module_filter | instmods
+    { find_kernel_modules_by_path drivers/net; find_kernel_modules_by_path drivers/s390/net } \
+        | net_module_filter | instmods
 
     instmods ecb arc4
     # bridge modules
diff --git a/modules.d/90multipath/module-setup.sh b/modules.d/90multipath/module-setup.sh
index f3b40bc..b78c005 100755
--- a/modules.d/90multipath/module-setup.sh
+++ b/modules.d/90multipath/module-setup.sh
@@ -45,7 +45,7 @@ installkernel() {
         done
     }
 
-    ( find_kernel_modules_by_path drivers/scsi drivers/s390/scsi ;
+    ( find_kernel_modules_by_path drivers/scsi; find_kernel_modules_by_path drivers/s390/scsi ;
       find_kernel_modules_by_path drivers/md )  |  mp_mod_filter  |  instmods
     [[ $debug ]] && set -x
 }
diff --git a/modules.d/95iscsi/module-setup.sh b/modules.d/95iscsi/module-setup.sh
index d645dba..4b7e292 100755
--- a/modules.d/95iscsi/module-setup.sh
+++ b/modules.d/95iscsi/module-setup.sh
@@ -52,7 +52,7 @@ installkernel() {
             esac
         done
     }
-    find_kernel_modules_by_path drivers/scsi drivers/s390/scsi \
+    { find_kernel_modules_by_path drivers/scsi; find_kernel_modules_by_path drivers/s390/scsi } \
     | iscsi_module_filter  |  instmods
 }
 
